### ISSUE TITLE
tidb-drainer cannot be deployed successfully out-of-box

### ISSUE THREAD

**Issue by shinnosuke-okada** [pingcap/tidb-operator#792]
## Bug Report

**What version of Kubernetes are you using?**
1.13

**What version of TiDB Operator are you using?**
1.0.0-Release1

**What storage classes exist in the Kubernetes cluster and what are used for PD/TiKV pods?**
N/A

**What's the status of the TiDB cluster pods?**
tidb-drainer pod crashes right after container start

**What did you do?**
Install TiDB cluster v2.1.16 using GA helm charts (v1.0.0)

**What did you expect to see?**
Successful drainer deployment out-of-box

**What did you see instead?**
Error due to incompatible changes made in drainer from v2.1.15 to v2.1.16
```
2019/08/20 13:38:10 main.go:26: [fatal] verifying flags error, See 'drainer --help'. component drainer's config file /etc/drainer/drainer.toml contained unknown configuration options: syncer.detect-interval, syncer.to.compression
github.com/pingcap/tidb-binlog/pkg/util.StrictDecodeFile
    /home/jenkins/workspace/release_tidb_2.1/go/src/github.com/pingcap/tidb-binlog/pkg/util/util.go:159
github.com/pingcap/tidb-binlog/drainer.(*Config).configFromFile
    /home/jenkins/workspace/release_tidb_2.1/go/src/github.com/pingcap/tidb-binlog/drainer/config.go:211
github.com/pingcap/tidb-binlog/drainer.(*Config).Parse
    /home/jenkins/workspace/release_tidb_2.1/go/src/github.com/pingcap/tidb-binlog/drainer/config.go:156
main.main
    /home/jenkins/workspace/release_tidb_2.1/go/src/github.com/pingcap/tidb-binlog/cmd/drainer/main.go:25
runtime.main
    /usr/local/go/src/runtime/proc.go:200
runtime.goexit
    /usr/local/go/src/runtime/asm_amd64.s:1337
```

**Comment by aylei** [comment]
What's your installation command?

**Comment by shinnosuke-okada** [comment]
@aylei it was like this:
`helm install http://download.pingcap.org/tidb-drainer-latest.tgz --name tidb-drainer --namespace ${CLUSTER_NAME}`

Sorry, I had a fix already but had to run to get dinner 😆I made a PR just now - please check. It was due to incompatible config format changes from drainer.

**Comment by aylei** [comment]
@shinnosuke-okada Thanks!

**Comment by aylei** [comment]
closed via #793
